package.json

```diff
@@ -7,7 +7,7 @@
     "build": "rm -rf dist/ && babel server/ -d dist/server/ && babel index.js -d dist/",
     "start": "export NODE_ENV=production&& node dist",
     "heroku-postbuild": "npm run migration && npm run seed && npm run build",
-    "test": "export NODE_ENV=test&& npm run migration && nyc mocha --require @babel/register tests/**/*.spec.js --timeout 5000 --exit",
+    "test": "export NODE_ENV=test&& npm run migration && nyc mocha --require @babel/register tests/**/*.spec.js --timeout 10000 --exit",
```

Comment:
Is there a reason why the timeout was increased?

Reply:
Yes. The test case for user profile image upload takes more than 5000ms (5secs) to complete. It was up to 7.3 secs on a particular occasion.